src/dev/src/app/checkboxes/checkboxes.demo.html

```diff
@@ -470,55 +470,3 @@ <h4>Compact</h4>
     </div>
   </div>
 </form>
-
-<h2>Validation in Groups (WIP)</h2>
```

Comment:
Any reason we removed this all together?

Reply:
I removed this because it was very unhappy with the type checking of the code, and we have it in git history to pull it back up some day.